tests/integration/integration_nodejs_test.go

```diff
@@ -801,7 +801,7 @@ func TestResourceWithSecretSerializationNodejs(t *testing.T) {
 	})
 }
 
-//nolint:paralleltest // mutates environment variables
+//nolint:paralleltest // Calls t.Setenv in the test
```

Comment:
The linter doesn't see this one?

Reply:
No seems it only sees things on the main path